### PR TITLE
Srs

### PR DESCRIPTION
**Pull request by fefrain**

### CODE REVIEW

File: common/app/srs.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import * as R from 'ramda'
```
Comment (minotogna) [inline common/app/srs.js]: in common you need to user require('ramda'). es6 imports do not work in node

File: common/app/srs.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+import codes from '@esri/proj-codes'
```
Comment (minotogna) [inline common/app/srs.js]: same as above

File: webapp/commonComponents/dropdownComponent.js [hunk 3]
```diff
@@ -85,16 +87,24 @@ class DropdownComponent extends React.Component {
...
 
     const contains = (a = '', b = '') => R.contains(R.toLower(a), R.toLower(b))
 
-    const filteredItems = R.trim(value).length > 0
-      ? items.filter(item => item.key
-        ? contains(value, item.key) || contains(value, item.value)
-        : contains(value, item)
+    const searchValue = R.trim(value)
+
+    let filteredItems
```
Comment (minotogna) [inline webapp/commonComponents/dropdownComponent.js]: here i would use ? : . i wouldn't use mutable objects with let

File: common/app/srs.js (hunk 1 above)
Comment (fefrain) [reply common/app/srs.js]: done

**Pull request merged**